pipeline_steps/common.groovy

```diff
@@ -968,7 +968,12 @@ void use_node(String label=null, body){
       print "${env.NODE_NAME} preparation complete, now ready for use."
       body()
     } catch (e){
-      print "Caught exception on ${env.NODE_NAME}: ${e}"
+      errString = "Caught exception on ${env.NODE_NAME}: ${e} Build: ${env.BUILD_URL}"
+      print errString
+
+      if (env.NODE_NAME =~ /^nodepool-/ && env.HOLD_ON_ERROR.asBoolean() == true){
```

Comment:
`isNodepoolNode` might be useful to add as a separate function.

Reply:
Done